### PR TITLE
[3/N] Refine beginner tutorial by accelerator api

### PR DESCRIPTION
**Pull request by guangyey**
For https://github.com/pytorch/tutorials/issues/3113

# Motivation
* __->__ #3170
* #3168
* #3167

As titled, we aim to use accelerator API to refine the beginner model tutorial to make it device-agnostic.

## Checklist

- [ ] The issue that is being fixed is referred in the description (see above "Fixes #ISSUE_NUMBER")
- [ ] Only one issue is addressed in this pull request
- [ ] Labels from the issue that this PR is fixing are added to this pull request
- [ ] No unnecessary issues are included into this pull request.

### CODE REVIEW

File: beginner_source/chatbot_tutorial.py [hunk 1]
```diff
@@ -1318,16 +1320,16 @@ def evaluateInput(encoder, decoder, searcher, voc):
     encoder_optimizer.load_state_dict(encoder_optimizer_sd)
     decoder_optimizer.load_state_dict(decoder_optimizer_sd)
 
-# If you have CUDA, configure CUDA to call
+# If you have accelerator, configure it to call
```
Comment (albanD) [inline beginner_source/chatbot_tutorial.py:1323]:
```suggestion
# If you have an accelerator, configure it to call
```

**Review by albanD — COMMENTED** [review]
Sounds good to me.
Will let @svekars take a final look.

File: beginner_source/chatbot_tutorial.py (hunk 1 above)
Comment (guangyey) [reply beginner_source/chatbot_tutorial.py]: Thanks @albanD
